README.md

```diff
@@ -7,7 +7,9 @@ This is the server for the Q Toolbox. To make use of Q server you will also need
 Documentation: https://nzzdev.github.io/Q-server
 
 ## Development
-Run tests `npm run test`
+### Tests
+There is a 100% coverage aim with the tests. Please do not lower the coverage.
```

Comment:
Not 80%?

Another question concerning tests: If I run it once it works and creates DB q-items, but on a second run this DB is still there and then tests fail because it tries to create it again. Should there be a change in the test script or do I have to do anything differently - and if yes it should be documented here I guess.

Reply:
there is now something logged to the console in case pouchdb-server is not killed at the end of the test. that should help.